src/imaginary/manipulation.py

```diff
@@ -50,7 +51,7 @@ def setIllumination(self, candelas):
 
 
 
-class Manipulator(Item):
+class Manipulator(Item, Enhancement):
```

Comment:
I think for MRO reasons this should be:
```suggestion
class Manipulator(Enhancement, Item):
```

Reply:
Hm interesting point.  There seems to be a lot of `Item, Enhancement` throughout the rest of the codebase.

I suspect there could be abstract as well as practical reasons to prefer one MRO over the other.  I don't see any methods implemented by both `Item` and `Enhancement` though ... Oh, unless `__init__` counts.  Only that seems to argue for putting `Item` first so `Item.__init__` beats `object.__init__`, doesn't it?  So I'm not sure what the practical reasons are.

Can you remind me of the abstract reasons?  Even a link to the explanation for me to read would be wonderful.  Thanks.

Reply:
The main thing putting `Enhancement` first does is allow it to override things from `Item`. As you point out, there are no such things right now, so it makes no difference unless such things are added in future. However, imagine a parallel universe where `Item` defines an empty `installed()` method; then having `Item` first would be broken.

Note that `object` is already in `Item`'s MRO so it ends up in the same place either way.